lib/sessions.js

```diff
@@ -549,7 +556,7 @@ class ServerSession {
       (((Date.now() - this.lastUse) % 86400000) % 3600000) / 60000
     );
 
-    return idleTimeMinutes > sessionTimeoutMinutes - 1;
+    return idleTimeMinutes > +sessionTimeoutMinutes - 1;
```

Comment:
this might just mean that `sessionTimeoutMinutes` is incorrectly documented above. Perhaps it should just be `Number`